lib/tasks/webpacker/clobber.rake

```diff
@@ -0,0 +1,17 @@
+require "webpacker/configuration"
+
+namespace :webpacker do
+  desc "Remove the webpack compiled output directory"
+  task clobber: ["webpacker:verify_install", :environment] do
+    output_path = Webpacker::Configuration.output_path
+    FileUtils.rm_r(output_path) if File.exist?(output_path)
+    puts "Removed webpack output path directory #{output_path}"
```

Comment:
Don't think we need to be chatty if this is working. Don't think assets:clobber says something on success, does it?